setup.cfg

```diff
@@ -27,6 +27,7 @@ dev =
     pydocstyle==6.1.1
     pylint==2.12.2
     pytest==7.0.0
+    pytest-black==0.3.12
```

Comment:
I like this! I also didn't know about `pytest-mypy`. This largely replaces me calling all of those things by themselves before calling `pytest` in any `test.sh`.